src/views/login/pronote/PronoteInstanceSelector.tsx

```diff
@@ -299,14 +333,17 @@ const PronoteInstanceSelector: Screen<"PronoteInstanceSelector"> = ({
                       );
                     }}
                     text={instance.name}
-                    subtext={`à ${instance.distance.toFixed(2)}km de toi`}
+                    subtext={
+                      prevRoute.name === "PronoteManualLocation"
+                        ? undefined
+                        : `à ${instance.distance.toFixed(2)}km de toi`
```

Comment:
Peut-être un peu trop stricte mais j'aime bien les majuscules 😅😅, non? 😂

```suggestion
                        : `À ${instance.distance.toFixed(2)}km de toi`
```

Et peut-être changer `de toi` => `de ta position` pour que ça soit plus français ? Vous en pensez quoi ?

Reply:
En soit c un sous titre qui a peu d'importance, même si je suis d'accord avec toi, à plusieurs endroits de l'application ils n'utilisent pas de majuscules...